### DISCUSSION TITLE
다크모드 적용 시 본문 글자 색상 오류

### DISCUSSION THREAD

**Discussion by chaewon9472** [pronist/hELLO#230] [Q&A] [answered]
다크모드로 들어가면 본문의 글자가 그대로 검정색을 유지하여 배경색과 같아져서 글자가 보이지 않습니다. 

제작자님 블로그에 적혀있듯이 복사 붙여넣기를 한 글은 에디터에서 타의적으로 인라인 스타일을 적용해서 색상이 강제되기 때문에 이미 작성된 글에 대해서는 <span style= "color:  "></span> 형태의 코드를 제거하라고 하셨는데 작성한 글이 많아서 저 방법 외에 더 간편한 방법이 있는 것인지 여쭙고 싶습니다. 

아래 첨부사진은 직접 적은 부분은 흰색으로 잘 뜨지만, 복붙하거나 복붙 후 일부 수정한 문장에 대해서는 검정색으로 뜨는 사진입니다.
https://avocadotori.tistory.com/142

![image](https://github.com/pronist/hello/assets/163621982/8a044504-17b2-4731-8ea4-6b4506700fd7)

**Comment by pronist** [comment, marked as answer]
이 문제에 대해서는 [스킨 배포 페이지](https://pronist.tistory.com/5#%EA%B8%B0%EB%8A%A5-1)에서 기능 - 다크모드에서 원인과 해결방법을 보시면 되겠습니다. 이 문제 해결을 위한 더 편한 방법은 제공하지 않습니다. 블로거의 자유도를 위해 티스토리 스킨 에디터에서 부여하는 것이므로 스킨에서 따로 건드리지 않습니다.

**Reply by pronist** [reply]
@chaewon9472 한 가지, CSS 에서 `!important` 로 본문의 색상을 강제로 변경하는 시도는 해볼 수 있습니다. 이 시도는 해보지를 않아서 확신은 없지만 해볼만할겁니다. 대신, 다크모드에서 에디터에서 색상을 변경하는 것은 무효화됩니다.

**Reply by chaewon9472** [reply]
답변 감사합니다!
